src/main/java/com/google/sps/servlets/InterviewerFeedbackServlet.java

```diff
@@ -111,33 +104,26 @@ private boolean interviewExists(long scheduledInterviewId) {
     return scheduledInterviewDao.get(scheduledInterviewId).isPresent();
   }
 
-  private boolean isInterviewee(long scheduledInterviewId, String userId) {
-    ScheduledInterview scheduledInterview = scheduledInterviewDao.get(scheduledInterviewId).get();
+  private ScheduledInterview getInterview(long scheduledInterviewId) {
+    return scheduledInterviewDao.get(scheduledInterviewId).get();
+  }
+
+  private boolean isInterviewee(ScheduledInterview scheduledInterview, String userId) {
     return scheduledInterview.intervieweeId().equals(userId);
   }
 
-  private String getInterviewerEmail(long scheduledInterviewId) {
-    ScheduledInterview scheduledInterview = scheduledInterviewDao.get(scheduledInterviewId).get();
+  private String getInterviewerEmail(ScheduledInterview scheduledInterview) {
     return personDao
         .get(scheduledInterview.interviewerId())
         .map(Person::email)
         .orElse("Email not found");
   }
 
-  private String getDateString(long scheduledInterviewId) {
-    ScheduledInterview scheduledInterview = scheduledInterviewDao.get(scheduledInterviewId).get();
-    TimeRange when = scheduledInterview.when();
-    LocalDateTime start = LocalDateTime.ofInstant(when.start(), ZoneId.systemDefault());
-    String startTime = start.format(DateTimeFormatter.ofPattern("h:mm a"));
-    String day = start.format(DateTimeFormatter.ofPattern("EEEE, MMMM d, yyyy"));
-    return String.format("%s at %s UTC", day, startTime);
-  }
-
   private void sendFeedback(String intervieweeEmail, HashMap<String, String> answers)
       throws IOException, Exception {
     EmailSender emailSender = new EmailSender(new Email("interviewme.business@gmail.com"));
     String subject = "Your Interviewee has submitted feedback for your interview!";
-    Email recipient = new Email(intervieweeEmail);
+    Email recipient = new Email("grantflash@gmail.com");
```

Comment:
?

Reply:
Forgot to remove the email I was using to test.